code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -628,3 +588,24 @@
 	item_state = "syndie_helm"
 	item_color = "syndi"
 	armor = list("melee" = 40, "bullet" = 50, "laser" = 30, "energy" = 15, "bomb" = 35, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 100)
+
```

Comment:
All the deleted stuff above this line needs to be re-added. Easiest way would probably be to just copy the shielded hardsuit code, reset the file to master, and paste.

Reply:
Fixed... I think